powerauth-nextstep/src/main/java/io/getlime/security/powerauth/app/nextstep/configuration/SecurityConfig.java

```diff
@@ -17,27 +17,30 @@
  */
 package io.getlime.security.powerauth.app.nextstep.configuration;
 
+import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
-import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.web.SecurityFilterChain;
 
 /**
  * Default Spring Security configuration.
  *
  * @author Petr Dvorak, petr@wultra.com
  */
 @Configuration
-public class SecurityConfig extends WebSecurityConfigurerAdapter {
+public class SecurityConfig {
```

Comment:
I believe that we should add `@EnableWebSecurity`.

Reply:
Sure, added.